.github/dependabot.yml

```diff
@@ -14,3 +14,13 @@ updates:
       interval: "daily"
     pull-request-branch-name:
       separator: "-"
+
+  - package-ecosystem: "gomod"
+    directory: "/actions/trigger-argo-workflow"
```

Comment:
Also for https://github.com/grafana/shared-workflows/tree/main/actions/techdocs-rewrite-relative-links please :)

Reply:
a suggestion would be nice heh